editor/state.js

```diff
@@ -34,6 +34,8 @@ export const editor = combineUndoableReducers( {
 					...state,
 					...action.edits,
 				};
+			case 'REQUEST_POST_UPDATE':
```

Comment:
I understand this concern. And this is part of defining what is our Data Approach, how do we handle async actions etc... We should document this and be consistent.